stan.cabal

```diff
@@ -101,16 +101,25 @@ executable stan
   main-is:             Main.hs
   build-depends:       stan
 
+library test-examples
+  import:              common-options
+  hs-source-dirs:      target
+  exposed-modules:     Target.HEAD
```

Comment:
Will we have a separate file per inspection? I was thinking of having them all together :thinking: 
However, I see how testing several modules can be useful as well, but at the moment I guess one is enough. Do you ming to rename it to something more general? :slightly_smiling_face:

Reply:
@vrom911 Good idea :+1: I guess we won't ever have modules per inspection, maybe at least by some category. Renaming to something like `Target.Example` sounds reasonable for now :+1: